src/auth.py

```diff
@@ -33,18 +33,19 @@ def register():
     return render_template("register.html")
 
 
-@auth.route("/register", methods = ["POST"])
+@auth.route("/register", methods=["POST"])
 def register_post():
     email = request.form.get("email")
     name = request.form.get("name")
     password = request.form.get("password")
 
-    user = User.query.filter_by(email = email).first()
+    user = User.query.filter_by(email=email).first()
 
     if user:
         return redirect(url_for("auth.register"))
 
-    new_user = User(email = email, name = name, password = generate_password_hash(password, method = "sha256"))
+    new_user = User(email=email, name=name,
```

Comment:
Jostain syystä mun autopep8 muuttaa näitä muotoiluja pikkasen jänniksi. Kaikki toimii kuitenkin normaalisti